mocks/firestore.js

```diff
@@ -298,13 +296,17 @@ FakeFirestore.DocumentReference = class {
     if (this._get().exists) {
       this.firestore._updateData(this.path, object, true);
     }
-    return Promise.resolve(buildDocFromHash({ ...object, _ref: this }));
+    return Promise.resolve(
+      buildDocFromHash({ ...object, _ref: this, _updateTime: timestamp.Timestamp.now() }),
```

Comment:
We have some asserts that test for the presence of `updateTime` and `readTime` but it would be nice if we could assert the logic here too.  Can we add an additional test cases just to assert a time is returned?

Reply:
Good catch @sbatson5 ! Will add the necessary test cases.